pkg/scheduler/framework/plugins/clusteraffinity/cluster_affinity.go

```diff
@@ -31,8 +31,12 @@ func (p *ClusterAffinity) Name() string {
 }
 
 // Filter checks if the cluster matched the placement cluster affinity constraint.
-func (p *ClusterAffinity) Filter(ctx context.Context,
-	bindingSpec *workv1alpha2.ResourceBindingSpec, cluster *clusterv1alpha1.Cluster) *framework.Result {
+func (p *ClusterAffinity) Filter(
+	_ context.Context,
+	bindingSpec *workv1alpha2.ResourceBindingSpec,
+	bindingStatus *workv1alpha2.ResourceBindingStatus,
+	cluster *clusterv1alpha1.Cluster,
+) *framework.Result {
```

Comment:
Why not finish the adoption here? That's the reason why introducing `binding status`.